TestCases/compliance-level-3/0055-feel-odd-function/0055-feel-odd-function.dmn

```diff
@@ -0,0 +1,158 @@
+    <decision name="decision002" id="_decision002">
+        <description>Tests FEEL expression: 'odd(-2)' and expects result: 'false (boolean)'</description>
+        <question>Result of FEEL expression 'odd(-2)'?</question>
+        <allowedAnswers>false (boolean)</allowedAnswers>
+        <variable typeRef="boolean" name="decision002"/>
+        <literalExpression>
+            <text>odd(-2)</text>
+        </literalExpression>
+    </decision>
+    <decision name="decision002_a" id="_decision002_a">
+        <description>Tests FEEL expression: 'odd(-1)' and expects result: 'true (number)'</description>
+        <question>Result of FEEL expression 'odd(-1)'?</question>
+        <allowedAnswers>true (boolean)</allowedAnswers>
+        <variable typeRef="boolean" name="decision002_a"/>
+        <literalExpression>
+            <text>odd(-1)</text>
+        </literalExpression>
+    </decision>
+    <decision name="decision003" id="_decision003">
+        <description>Tests FEEL expression: 'odd(0)' and expects result: '0 (number)'</description>
+        <question>Result of FEEL expression 'odd(0)'?</question>
+        <allowedAnswers>0 (number)</allowedAnswers>
+        <variable typeRef="boolean" name="decision003"/>
+        <literalExpression>
+            <text>odd(0)</text>
+        </literalExpression>
+    </decision>
+    <decision name="decision004" id="_decision004">
+        <description>Tests FEEL expression: 'odd()' and expects result: 'null (number)'</description>
+        <question>Result of FEEL expression 'odd()'?</question>
+        <allowedAnswers>null (number)</allowedAnswers>
+        <variable typeRef="boolean" name="decision004"/>
+        <literalExpression>
+            <text>odd()</text>
+        </literalExpression>
+    </decision>
+    <decision name="decision005" id="_decision005">
+        <description>Tests FEEL expression: 'odd(4,4)' and expects result: 'null (number)'</description>
+        <question>Result of FEEL expression 'odd(4,4)'?</question>
+        <allowedAnswers>null (number)</allowedAnswers>
+        <variable typeRef="boolean" name="decision005"/>
+        <literalExpression>
+            <text>odd(4,4)</text>
+        </literalExpression>
+    </decision>
+    <decision name="decision006" id="_decision006">
+        <description>Tests FEEL expression: 'odd(number:4)' and expects result: '2 (number)'</description>
+        <question>Result of FEEL expression 'odd(number:4)'?</question>
+        <allowedAnswers>2 (number)</allowedAnswers>
+        <variable typeRef="boolean" name="decision006"/>
+        <literalExpression>
+            <text>odd(number:4)</text>
+        </literalExpression>
+    </decision>
+    <decision name="decision007" id="_decision007">
+        <description>Tests FEEL expression: 'odd(n:4)' and expects result: 'null (number)'</description>
+        <question>Result of FEEL expression 'odd(n:4)'?</question>
+        <allowedAnswers>2 (number)</allowedAnswers>
```

Comment:
from ` <allowedAnswers>2 (number)</allowedAnswers>`
to ` <allowedAnswers>null (boolean)</allowedAnswers>`

Reply:
@tarilabs amended.